python/orca/src/bigdl/orca/learn/tf2/spark_runner.py

```diff
@@ -297,13 +306,13 @@ def setup(self):
         os.environ["KMP_BLOCKING_TIME"] = self.config.get("KMP_BLOCKING_TIME",
                                                           os.environ.get("KMP_BLOCKING_TIME", "0"))
 
-    def setup_local(self):
+    def setup_local(self, custom_objects, compile):
```

Comment:
Maybe modify the function to `def setup_local(self)`, and use class member variables in function